sw/otbn/crypto/tests/p384_ecdsa_verify_test.s

```diff
@@ -135,6 +116,6 @@ pub_y:
   .word 0xaaafcad2
   .zero 16
 
-/* signature verification result x_res */
-sig_xres:
+/* signature verification result x_res (rnd) */
+rnd:
   .zero 64
```

Comment:
This bit is a little confusing to me -- why is the signature verification result called `rnd`? When I search for it in the code, it only ever seems to be used for the returned x-coordinate. I'd call it maybe `x_r`, to match the p256 code.

Reply:
I don't know why it's called `rnd`. I just stuck to the naming in `p384_verify.s`, where it's also named `rnd`. This was already there before I started working on the code. But tbh it confused me too. Maybe it's the right moment now to change this to `x_r` throughout the code.